README.md

```diff
@@ -12,144 +12,361 @@
     </div>
 </div>
 
-## Workflow is a Golang workflow framework that encompasses these main features:
-- Defining small units of work called "Steps"
-- Consumer management and graceful shutdown
-- Supports event streaming platforms such as Kafka and Reflex (or you can write your own implementation of the EventStreamer interface!)
-- Built in support for timeout operations (e.g. account cool down periods etc).
-- Built in support for callbacks (e.g. Call an async endpoint and trigger the callback from a webhook handler).
-- Connect two workflows together. Wait for specific events from another workflow and make it part of your workflow!
-- Super Duper testable
-
+# Workflow
+ Workflow is an event driven workflow that allows for robust, durable, and scalable sequential business logic to
+ be executed in a deterministic manner.
+---
+## Features
+
+- **Tech stack agnostic:** Use Kafka, Cassandra, Redis, MongoDB, Postgresql, MySQL, RabbitM, or Reflex - the choice is yours!
+- **Graph based (Directed Acyclic Graph - DAG):** Design the workflow by defining small units of work called "Steps".
+- **TDD:** Workflow was built using TDD and remains well-supported through a suit of tools.
+- **Schedule:** Allows standard cron spec to schedule workflows 
+- **Timeouts:** Set either a dynamic or static time for a workflow to wait for. Once the timeout finishes everything continues as it was.
+- **Event fusion:** Add event connectors to your workflow to consume external event streams (even if its from a different event streaming platform).  
+- **Callbacks:** Allow for manual callbacks from webhooks or manual triggers from consoles to progress the workflow such as approval buttons or third-party webhooks.  
+- **Parallel consumers:** Specify how many step consumers should run or specify the default for all consumers. 
+- **Consumer management:** Consumer management and graceful shutdown of all processes making sure there is no goroutine leaks!
+
+---
+
+## Installation
+To start using workflow you will need to add the workflow module to your project. You can do this by running:
+```go
```

Comment:
Think these should be shell, as they'll be run from terminal, and aren't Go code (colouring doesn't look right as a result)
![image](https://github.com/user-attachments/assets/e947f872-ff3d-40c8-9c3c-350a9efd2779)


```suggestion
```shell
```

Reply:
oh silly me. Good catch